src/cache/core/cache.ts

```diff
@@ -141,6 +174,46 @@ export abstract class ApolloCache<TSerialized> implements DataProxy {
     });
   }
 
+  public watchFragment<TData = any, TVars = OperationVariables>(
```

Comment:
```suggestion
  public watchFragment<TData = unknown, TVars = OperationVariables>(
```

For new APIs, I think we should default to `unknown` on the data type for a bit better type safety.

Reply:
Updated in https://github.com/apollographql/apollo-client/pull/11465/commits/67c1e1cb18b242644a36b832607b2abaea4a6714